website/src/content/api-reference/application-config.mdx

```diff
@@ -327,7 +327,7 @@ You can have "view-only" or "manage-only" OAuth Scopes and leave the other list
 
 ## `oAuthScopes.view`
 
-A list of "view-only" [OAuth Scopes](https://docs.commercetools.com/api/scopes) required by the Custom Application and associated to the `View` permission.
+A list of "view-only" [OAuth Scopes](https://docs.commercetools.com/api/scopes) required by the Custom Application and associated with the `View` permission.
```

Comment:
I would prefer the documentation changes to be defined in a separate PR. Main reason being that merging this PR will publish the docs changes but the packages are not being released yet.

So the docs changes can be merged separately once the packages have been released.

Can you extract these docs changes into a separate PR? Thanks 🙂